pyins/tests/test_util.py

```diff
@@ -3,6 +3,24 @@
 from pyins import util
 
 
+def test_mm_prod():
+    np.random.seed(0)
+    a = np.random.randn(3, 4)
+    b = np.random.randn(4, 3)
+    assert_allclose(util.mm_prod(a, b), a @ b)
+    assert_allclose(util.mm_prod(a, b, at=True, bt=True), a.T @ b.T)
```

Comment:
As I see it `@` now implements all the necessary broadcasting, so `util.mm_prov` and `mv_prod` (probably) should use it instead of `einsum`.